build.gradle

```diff
@@ -9,8 +9,7 @@ plugins {
   id "org.sonarqube" version "2.8"
 }
 
-// TODO: Update group to end with 'admin' or 'trainee'.
-group = 'uk.nhs.hee.tis'
+group = 'uk.nhs.hee.tis.trainee.forms'
```

Comment:
Remove `.forms` from the group, this is different from your standard base package and should instead reflect the same sort of grouping as in Maven: https://github.com/Health-Education-England/TIS-TCS/blob/master/pom.xml#L6

Reply:
So it is a groupId of maven. Nice to know. Thanks

Reply:
removed